### PR TITLE
Ddncls 5427

### PR DESCRIPTION
**Pull request by matthew-mcnally**

### CODE REVIEW

File: test/Controllers/StatusInfoControllerTest.scala [hunk 1]
```diff
@@ -49,7 +47,7 @@ class StatusInfoControllerTest extends UnitSpec with OneServerPerSuite with Mock
   "For API 11, Status Info Controller " should {
 
     "use the correct status info service" in {
-      StatusInfoController.statusInfoService shouldBe EtmpStatusInfoService
+      //StatusInfoController.statusInfoService shouldBe EtmpStatusInfoService
```
Comment (danewatts) [inline test/Controllers/StatusInfoControllerTest.scala]: Delete this test

Comment (danewatts) [inline test/Controllers/StatusInfoControllerTest.scala]: Delete this test

Comment (matthew-mcnally) [reply test/Controllers/StatusInfoControllerTest.scala]: done

File: test/Controllers/SubscriptionControllerTest.scala [hunk 2]
```diff
@@ -70,18 +67,12 @@ class SubscriptionControllerTest extends UnitSpec with OneServerPerSuite with Mo
...
-      }
-
-      "return text/plain" in {
         when(mockSubcriptionService.subscribe(Matchers.any(), Matchers.any(), Matchers.any(), Matchers.any(), Matchers.any())(Matchers.any())).thenReturn(Future.successful(registerSuccessResponse))
         val result = TestSubscriptionController.subscribe("").apply(FakeRequest().withJsonBody(api4FrontendLTDJson))
         contentType(result).get shouldBe "text/plain"
       }
 
-      "return Businessdetails for successful register" in {
+      "return Businessdetails for succSubscriptionController.subscribe(\"\").apply(FakeRequest().withJsonBody(api4FrontendLTDJson))\n        status(result) shouldBe OK\n      }\n\n      \"return text/plain\" in {\n        when(mockSubcriptionService.subscribe(Matchers.any(), Matchers.any(), Matchers.any(), Matchers.any(), Matchers.any())(Matchers.any())).thenReturn(Future.successful(registerSuccessResponse))\n        val result = TestSubscriptessful register" in {
```
Comment (daniel-manning) [inline test/Controllers/SubscriptionControllerTest.scala]: you've messed up the test name

File: test/Controllers/SubscriptionControllerTest.scala [hunk 3]
```diff
@@ -70,18 +67,12 @@ class SubscriptionControllerTest extends UnitSpec with OneServerPerSuite with Mo
       val matchFailureResponse = HttpResponse(NOT_FOUND, responseJson = Some(matchFailure))
 
       "respond with OK" in {
-        when(mockSubcriptionService.subscribe(Matchers.any(), Matchers.any(), Matchers.any(), Matchers.any(), Matchers.any())(Matchers.any())).thenReturn(Future.successful(registerSuccessResponse))
```
Comment (daniel-manning) [inline test/Controllers/SubscriptionControllerTest.scala]: why removed?

File: test/Controllers/StatusInfoControllerTest.scala [hunk 4]
```diff
@@ -48,10 +46,6 @@ class StatusInfoControllerTest extends UnitSpec with OneServerPerSuite with Mock
 
   "For API 11, Status Info Controller " should {
 
-    "use the correct status info service" in {
-      StatusInfoController.statusInfoService shouldBe EtmpStatusInfoService
```
Comment (daniel-manning) [inline test/Controllers/StatusInfoControllerTest.scala]: why removed?

File: test/Controllers/SubscriptionControllerTest.scala (hunk 2 above)
Comment (matthew-mcnally) [reply test/Controllers/SubscriptionControllerTest.scala]: done

File: test/Controllers/StatusInfoControllerTest.scala (hunk 4 above)
Comment (danewatts) [reply test/Controllers/StatusInfoControllerTest.scala]: I recommended removing this as its a test check the object being used is the correct implementation for that trait. However, this was causing complications and is an unnecessary test.

Comment (matthew-mcnally) [reply test/Controllers/StatusInfoControllerTest.scala]: done

**Pull request merged**